RecoTauTag/RecoTau/python/PATTauDiscriminationAgainstElectronMVA6Phase2_cff.py

```diff
@@ -0,0 +1,114 @@
+import FWCore.ParameterSet.Config as cms
+
+# Electron collection merger
+mergedSlimmedElectronsForTauId = cms.EDProducer('PATElectronCollectionMerger',
+    src = cms.VInputTag('slimmedElectrons', 'slimmedElectronsFromMultiCl')
+)
+# anti-e phase-2 tauID (Raw)
+from RecoTauTag.RecoTau.TauDiscriminatorTools import noPrediscriminants
+from RecoTauTag.RecoTau.patTauDiscriminationAgainstElectronMVA6_cfi import *
+patTauDiscriminationAgainstElectronMVA6Phase2Raw = patTauDiscriminationAgainstElectronMVA6.clone(
+    #Note: PATTauProducer has to be set in the final cfg
+    Prediscriminants = noPrediscriminants,
+    srcElectrons = "mergedSlimmedElectronsForTauId",
+    isPhase2 = True,
+    vetoEcalCracks = False,
+    mvaName_NoEleMatch_woGwoGSF_BL = "RecoTauTag_antiElectronPhase2MVA6v1_gbr_NoEleMatch_woGwoGSF_BL",
+    mvaName_NoEleMatch_wGwoGSF_BL = "RecoTauTag_antiElectronPhase2MVA6v1_gbr_NoEleMatch_wGwoGSF_BL",
```

Comment:
this and below apparently repeats a lot of what's in RecoTauTag/RecoTau/python/PFRecoTauDiscriminationAgainstElectronMVA6Phase2_cff.py
Some common source should be defined to provide a consistent configuration.
Also, IIUC, all values (including categories) are also encoded in loadRecoTauTagMVAsFromPrepDB_cfi.py

Reply:
OK, a common source (a cff) will be set.